cli/js/lib.deno.ns.d.ts

```diff
@@ -1315,6 +1373,7 @@ declare namespace Deno {
     UnexpectedEof: ErrorConstructor;
     BadResource: ErrorConstructor;
     Http: ErrorConstructor;
+    AssertionError: ErrorConstructor;
```

Comment:
This feels wrong, there will be type information about `actual` and `expected` 🤔